### PR TITLE
Optimized svg inlining

### PR DESCRIPTION
**Pull request by sergcen**
Replacing some encoded characters in inlined SVG. It's reduces encoded data for SVG.
gzip optimization: convert encoded characters to lower case.

### CODE REVIEW

File: lib/freeze.js [hunk 1]
```diff
@@ -40,6 +40,8 @@ const contentTypes = {
     '.woff2': 'application/font-woff2'
 };
 
+const inlineTypes = [':base64:', ':encodeURI:', ':encodeURIComponent:', ':encodeURIComponentSvg:'];
```
Comment (tadatuta) [inline lib/freeze.js]: it's better to switch to hash to make O(1) lookup.
and let's stick to `var` for consistency reason.

**Comment by tadatuta** [comment]
please provide additional description about the task

**Comment by sergcen** [comment]
@tadatuta fixed

**Comment by tadatuta** [comment]
@sergcen
LGTM but there's falling test. I know it's not your fault but I want to fix it first before merge.
Tell me if this update is urgent.

**Comment by tadatuta** [comment]
@sergcen will you please rebase the PR on top of current state of `master` to fix the tests?

**Pull request merged**